.github/CODEOWNERS

```diff
@@ -0,0 +1,37 @@
+# =================================================================================================
+# Files or directories with designated code owner.
+#
+# Note:
+#   - Each line is a file pattern followed by one or more owners.
+#   - Branch with branch protection rule "Require review from Code Owners"
+#     will automaticaly trigger a request to a designated code owner
+# =================================================================================================
+
+# ....Repository wide code owner...................................................................
+# These owners will be the default owners for everything in the repo.
+#*       @pomerlef
+
+
+# ....High level files.............................................................................
+/README.md @pomerlef
+/LICENSE @pomerlef
+
+# ....Core directories and files...................................................................
+/doc @boxanm @simonpierredeschenes @aguenette @pomerlef
+/pointmatcher @boxanm @simonpierredeschenes @aguenette
```

Comment:
Not sure what is the purpose of this file, but this list doesn't list ok complet

Reply:
@pomerlef The `CODEOWNERS` file designate who can approve a pull request on a protected branch for a given file/directories. They will also be notified when change are performed on those file/directories eg any change in the `pointmatcher` directory and child will trigger a code review request to the designated code owner.
I've use the name of those that had performed code review recently as a starting point.